target/start-mailserver.sh

```diff
@@ -170,8 +170,8 @@ function register_functions
   _register_setup_function "_setup_environment"
   _register_setup_function "_setup_logrotate"
 
-  [[ ${PFLOGSUMM_TRIGGER} != "none" ]] && _register_setup_function "_setup_mail_summary"
-  [[ ${LOGWATCH_TRIGGER} != "none" ]] && _register_setup_function "_setup_logwatch"
+  [[ ${PFLOGSUMM_TRIGGER:="none"} != "none" ]] && _register_setup_function "_setup_mail_summary"
```

Comment:
This change does not respect the way the `start-mailserver.sh`-script works. We setup a map `DEFAULT_VARS` which is populated with these variables. You will need to have a look at lines 398-405 too. The tests are failing due to `pflogsum` not being happy. You will need to implement your changes with `REPORT_RECIPIENT` (see the lines I mentioned), or apply for deprecated variables in `env-mailserver` to be removed, which must be done eventually, but would mean more work. If you'd like to know what is most likely best suite  here, I'd ask @erik-wramner.

Reply:
Hey, thanks very much for the comment and review, I will look at the lines you mention and make some changes soon to try to fix the REPORT variables like this if needed. I think I see the mistake that is failing the test, I accidentally assigned the value "none" to the  variables instead of just using it as a default. Hopefully the tests will pass now.